portal/src/app/components/home/components/experiment/components/charts/components/dynamic-chart/models/dynamic-chart.interface.ts

```diff
@@ -0,0 +1,20 @@
+import { AttSelectOption } from "../../../../../../../../../shared/components/att-select";
+
+export enum ChartType {
+    LINE = 'line',
+    BAR = 'bar',
+}
+
+export const chartTypeOptions: AttSelectOption[] = Object.keys(ChartType).map((key) => ({
+    value: ChartType[key as keyof typeof ChartType],
+    label: key,
+}));
+
+export enum XAxisType {
+   NUMBER_OF_ITERATIONS = 'number of iterations',
```

Comment:
Please change it to "Number of Iterations" like the Figma:
```suggestion
   NUMBER_OF_ITERATIONS = 'Number of Iterations',
```